composer.json

```diff
@@ -26,11 +22,19 @@
 			"url": "./packages/*"
 		}
 	],
+	"minimum-stability": "dev",
```

Comment:
Is this change in minimum stability required? we worked with `dev-master` already without this requiring to lower the stability here. consequence of it seems to be that all dependencies also get downloaded in development version. not sure if you want achieve that.

Reply:
Without it I get the following error:

```
  Problem 1
    - Root composer.json requires typo3/cms-introduction dev-master -> satisfiable by typo3/cms-introduction[dev-master].
    - typo3/cms-introduction dev-master requires bk2k/bootstrap-package dev-BP_12-0 -> found bk2k/bootstrap-package[dev-BP_12-0, 12.0.x-dev (alias of dev-BP_12-0)] but it does not match your minimum-stability.
```

Reply:
Got it removed